lib/types.nix

```diff
@@ -13,73 +15,72 @@ let
       merge = lib.options.mergeEqualOption;
     };
 in
-rec {
-  isRawType = v: lib.isAttrs v && lib.hasAttr "__raw" v && lib.isString v.__raw;
+  rec {
+    isRawType = v: lib.isAttrs v && lib.hasAttr "__raw" v && lib.isString v.__raw;
 
-  rawLua = mkOptionType {
-    name = "rawLua";
-    description = "raw lua code";
-    descriptionClass = "noun";
-    merge = mergeEqualOption;
-    check = isRawType;
-  };
+    rawLua = mkOptionType {
+      name = "rawLua";
+      description = "raw lua code";
+      descriptionClass = "noun";
+      merge = mergeEqualOption;
+      check = isRawType;
+    };
 
-  maybeRaw = type: types.either type rawLua;
+    maybeRaw = type: types.either type rawLua;
 
-  border =
-    with types;
-    oneOf [
-      str
-      (listOf str)
-      (listOf (listOf str))
-    ];
+    border = with types;
+      oneOf [
+        str
+        (listOf str)
+        (listOf (listOf str))
+      ];
 
-  logLevel = types.enum [
-    "off"
-    "error"
-    "warn"
-    "info"
-    "debug"
-    "trace"
-  ];
+    logLevel = types.enum [
+      "off"
+      "error"
+      "warn"
+      "info"
+      "debug"
+      "trace"
+    ];
```

Comment:
This kind of multi-line list was one of @GaetanLepage's biggest complaints with nixfmt, however alejandra doesn't appear to be any different :confounded: